### ISSUE TITLE
Responsive font changes result in flickering

### ISSUE THREAD

**Issue by Vaguery** [poole/lanyon#71]
[The demo site](http://lanyon.getpoole.com) is set up to detect media widths in rem, and respond by changing the font size.

At the default font size (I suppose 16px, in Safari?), whenever the window is between about 600px and 760px, the layout flickers. For other browser font sizes (for instance, zooming the typeface), the exact values are slightly different... but the result is that the response triggers its own undoing.

In other words, the font size reduces (and the line length grows), and then the font size grows (and the line length reduces)... and so on, forever.

**Comment by andrewcodes** [comment]
I'm seeing the same issue on iOS 8.1.3 with Chrome and Safari. Mobile WebKit apparently doesn't like rem sizing yet on mobile.

**Comment by andrewcodes** [comment]
Also, the Poole and Hyde themes don't have this issue on mobile.

**Comment by mdo** [comment]
Master fixed this—after I moved from `rem`s to `em`s in the media queries. Will have the demo site updated soon.
